### PR TITLE
Add optional input check to postgresql_ext

### PR DESCRIPTION
**Pull request by andytom**
##### SUMMARY
Related to #106 and #210

postgresql_ext:
 
* add trust_input parameter

##### ISSUE TYPE
* Feature Pull Request
 
##### COMPONENT NAME
`plugins/modules/database/postgresql/postgresql_ext.py`

### CODE REVIEW

File: tests/integration/targets/postgresql_ext/tasks/postgresql_ext_initial.yml [hunk 1]
```diff
@@ -161,7 +187,21 @@
...
 - assert:
     that:
     - result.rowcount == 1
   when: postgres_version_resp.stdout is version('9.6', '<=')
+
+- name: postgresql_ext - check that using a dangerous name fails
+  postgresql_ext:
+    db: postgres
+    name: postgis
+    session_role: 'curious.anonymous"; SELECT * FROM information_schema.tables; --'
```
Comment (Andersson007) [inline tests/integration/targets/postgresql_ext/tasks/postgresql_ext_initial.yml:200]:
```suggestion
    session_role: 'curious.anonymous"; SELECT * FROM information_schema.tables; --'
    trust_input: no
```

**Comment by Andersson007** [comment]
seems to be fully completed, LGTM

File: tests/integration/targets/postgresql_ext/tasks/postgresql_ext_initial.yml (hunk 1 above)
Comment (andytom) [reply tests/integration/targets/postgresql_ext/tasks/postgresql_ext_initial.yml:200]: Doh, I knew I missed something.

**Comment by andytom** [comment]
ready_for_review

**Comment by Andersson007** [comment]
@andytom thanks for the contribution!
merged into master

**Pull request merged**